server/routes/regionRouter.js

```diff
@@ -0,0 +1,67 @@
+const express = require('express');
+const helper = require('./utils');
+const Session = require('../models/Session');
+const RegionService = require('../services/RegionService');
+
+const regionRouter = express.Router();
+
+regionRouter.get(
+  '/',
+  //   helper.apiKeyHandler,
+  helper.handlerWrapper(async (req, res, next) => {
```

Comment:
I think in our modern APIs we have the handlers (handler logic) in a separate file and folder from the router.  Check server/routes.js in the treetracker-api

Reply:
Updated the structure to place the handler logic in a seperate folder and file.